roles/config-wireguard/config/sjc.yml

```diff
@@ -575,6 +575,19 @@ wg_peers:
     mp_bgp: true
     extended_next_hop: true
 
+- name: dn42-4242420057
+  port: 20057
+  remote: home.tompkel.net:13244
```

Comment:
It looks like a different server ([yvr](https://yvr.peer.highdef.network/webtrace/?action=mtr&target=home.tompkel.net)) is closer to you: ~8ms instead of ~20ms from sjc. Please create the peering there instead

Reply:
Sure, I've pushed a commit to switch this to yvr